roles/generate-jenkins/templates/DOCUMENTATION.j2

```diff
@@ -1,7 +1,8 @@
 ---
 title: {{ project_name }}
 ---
-# [{{ lsio_project_name_short }}/{{ project_name }}]({{ lsio_github_url }}/{{ project_repo_name }})
+<!-- DO NOT EDIT THIS FILE MANUALLY  -->
+<!-- Please read the {{ lsio_github_asset }}/.github/CONTRIBUTING.md -->
```

Comment:
Do we not need an h1 here? (the bit that you removed)

Reply:
The 3 lines above are a valid replacement in the linter-rules(and used by gitbook, according to their docs). The link is voided on gitbooks side(looking at the live sites).